tests/pipelines/test_config_contrastive.py

```diff
@@ -0,0 +1,188 @@
+                                    "transform_original": transform_original,
+                                },
+                            },
+                        },
+                        "callbacks": {
+                            "criterion": {
+                                "_target_": "CriterionCallback",
+                                "input_key": "projection_left",
+                                "target_key": "projection_right",
+                                "metric_key": "loss",
+                            },
+                            "sklearn_model": {
+                                "_target_": "catalyst.dl.SklearnModelCallback",
+                                "feature_key": "embedding_left",
+                                "target_key": "target",
+                                "train_loader": "train",
+                                "valid_loaders": "valid",
+                                "model_fn": "ensemble.RandomForestClassifier",
+                                "predict_method": "predict_proba",
+                                "predict_key": "sklearn_predict",
+                                "random_state": 42,
+                                "n_estimators": 10,
+                            },
+                            "accuracy": {
+                                "_target_": "catalyst.dl.ControlFlowCallback",
+                                "base_callback": {
+                                    "_target_": "catalyst.dl.AccuracyCallback",
+                                    "target_key": "target",
+                                    "input_key": "sklearn_predict",
+                                    "topk_args": [1, 3],
+                                },
+                                "loaders": "valid",
+                            },
+                            "optimizer": {"_target_": "OptimizerCallback", "metric_key": "loss",},
+                            "scheduler": {"_target_": "SchedulerCallback"},
+                            "checkpointer": {
+                                "_target_": "CheckpointCallback",
+                                "logdir": logdir,
+                                "loader_key": "train",
+                                "metric_key": "loss",
+                                "minimize": True,
+                                "save_n_best": 1,
+                            },
+                        },
+                    },
+                },
+            },
+        )
+
+        runner.run()
+
+        metrics_path = Path(logdir) / "_metrics.json"
+        with open(metrics_path, "r") as file:
+            metrics = json.load(file)
+
+        assert metrics["best"]["valid"]["accuracy"] > 0.6
+
+
+# Torch
+@mark.skipif(
+    not SETTINGS.ml_required or not SETTINGS.cv_required,
+    reason="catalyst[ml] and catalyst[cv] required",
+)
+def test_contrastive_on_cpu():
+    train_experiment({"_target_": "DeviceEngine", "device": "cpu"})
```

Comment:
could you please add the GPU test also? for the colab ci-cd

Reply:
[Done](https://github.com/gr33n-made/catalyst/blob/simCLR/tests/pipelines/test_config_contrastive.py#L182-L279)